frontend/src/components/client/models/getClientFeature.tsx

```diff
@@ -43,6 +43,15 @@ const getFilteredClients = (config: ClientFeatureBuilderConfig): ClientModel[] =
   return clients;
 }
 
+const getClientInvoices = (client: ClientModel, config: ClientFeatureBuilderConfig): InvoiceModel[] => {
+  let clientInvoices = config.invoices.filter(i => i.client._id === client._id);
+  if (config.filters.years && config.filters.years.length) {
+    clientInvoices = clientInvoices.filter(i => config.filters.years.includes(i.date.year()));
+  }
+  console.log('getClientInvoices', clientInvoices)
```

Comment:
console.log

Check: https://itenium.be/blog/productivity/git-pull-request-hygiene/